modules/github/README.md

```diff
@@ -15,6 +15,7 @@ Config explanations:
       - credentials:
             - oauthtoken: a Github OAuth2 or Personal Access Token
       - parameters:
+            - useldapgithubaccountmapping: indicates whether to use the Github field in LDAP as opposed to the uidmap in config to map LDAP usernames to Github usernames
```

Comment:
Yeah, I erred on the side of caution - didn't introduce divergent functionality right off the bat. I agree. Fixed.